integr8ly/integreatly-operator
# Bump Keycloak operator version to 12.0.3

# Description
https://issues.redhat.com/browse/INTLY-10442
Probably should be merged after https://github.com/keycloak/keycloak-operator/pull/313.

Steps to test the changes:
1. Install RHOAM  from master using the Integreatly operator.
2. Verify that Keycloak operator 12.0.1 was installed. Use e.g.:
```
oc get rhmis rhoam -n redhat-rhoam-operator -o=jsonpath="{.status.stages.products.products.rhssouser.operator}"
oc get deployments keycloak-operator -n redhat-rhoam-rhsso-operator -o=jsonpath="{.spec.template.spec.containers[0].image}"
```
3. Stop the operator and run it again from this PR.
4. Verify Keycloak operator was updated to 12.0.3 using the steps from 2.

## Type of change




## Checklist
